cli/pcluster/config/validators.py

```diff
@@ -783,6 +761,29 @@ def s3_bucket_validator(param_key, param_value, pcluster_config):
     return errors, warnings
 
 
+def fsx_lustre_auto_import_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if param_value != "NONE":
```

Comment:
Can you add a check for `None` here as well? `if param_value and param_value != "NONE"`

Strictly speaking, it shouldn't be necessary. The `auto_import_policy` parameter's entry in the `FSX` dict in mappings.py doesn't specify a `type`, so it defaults to the section's type's (`CfnSection`) default parameter type, `CfnParam`. This class uses the base `Param` class's validate implementation, which does check for `None` values.

There's a lot that could be changed there, which is the motivation for being more defensive in these validators with respect to uninitialized values.